### PR TITLE
updates env to pick up service ID

### PR DESCRIPTION
**Pull request by invincibleJai**
Signed-off-by: invinciblejai <mailjai.vardhan@gmail.com>

### CODE REVIEW

**Comment by invincibleJai** [comment]
@msrb `service id` will go as env var , had discussed on this with @samuzzal-choudhury

File: openshift/template.yaml [hunk 1]
```diff
@@ -132,3 +134,8 @@ parameters:
   required: true
   name: GUNICORN_CMD_ARGS
   value: "-b :5000 --workers=4"
+
+- description: "Subset of services to run. Use comma separated list of service ids (eg. 42,1337)"
+  value:
+  name: SERVICES_LIST
```
Comment (msrb) [inline openshift/template.yaml]: And just to be 200% sure, you're comfortable having those service IDs in https://github.com/openshiftio/saas-analytics/blob/master/bay-services/3scale-admin-gateway.yaml, where anybody can see them - so they are not secrets. Just asking, please confirm @samuzzal-choudhury @invincibleJai, thanks 😉

Comment (invincibleJai) [reply openshift/template.yaml]: @msrb we should be good with that we already have route and token as secrets 
CC @samuzzal-choudhury

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
CI works, at least for PRs... :)

File: openshift/template.yaml (hunk 1 above)
Comment (invincibleJai) [reply openshift/template.yaml]: @msrb this env i have kept optional , looking into ways so that we can avoid it

**Comment by invincibleJai** [comment]
closing it as https://github.com/fabric8-analytics/f8a-3scale-connect-api/pull/31 refers

**Pull request closed without merging**